nyc_data/ppe/tests.py

```diff
@@ -50,6 +58,18 @@ def test_mayoral_rollup(self):
         self.assertEqual(rollup[dc.MayoralCategory.iso_gowns],
                          AssetRollup(asset=dc.MayoralCategory.iso_gowns, demand=20, sell=5))
 
+    def test_only_aggregate_active_items(self):
+        self.data_import.status = ImportStatus.replaced
+        self.data_import.save()
+        try:
+            rollup = aggregations.asset_rollup(
+                datetime.now() - timedelta(days=28), datetime.now()
+            )
+            self.assertEqual(rollup[dc.Item.gown], AssetRollup(asset=dc.Item.gown, demand=0, sell=0))
```

Comment:
Don't think we have to spend time on this right now, but am confused about how the returned value of `asset_rollup` from the aggregations file is equal to an instantiated version of the `AssetRollup` class. 

Doesn't seem urgent, but wanted to log now to remind myself to ask you about it at some point.

Reply:
It isn't exactly -- it's a dict[str, rollup]